wmn-data.json

```diff
@@ -5781,6 +5901,18 @@
         "cat" : "gaming",
         "valid" : true
        },
+       {
+        "name" : "toyhou.se",
+        "uri_check" : "https://toyhou.se/{account}",
+        "post_body" : "",
+        "e_code" : 200,
+        "e_string" : "bulletins",

```

Comment:
Please swap out this simple word with something more complex/specific like `display-user`